test/extension/generalMobilePlatform.test.ts

```diff
@@ -29,5 +30,38 @@ suite("generalMobilePlatform", function () {
                 assert.equal(GeneralMobilePlatform.getOptFromRunArgs(args, "param4", false), undefined);
             });
         });
+
+        suite("getEnvArgument", function() {
+            let origEnv: any = {"test1": "origEnv", "test2": "origEnv", "test3": "origEnv"};
+            let env: any = {"test2": "env", "test3": "env", "test4": "env"};
+            let envForFile: string = "test3=envFile\ntest4=envFile\ntest5=envFile";
+            let envFile: string = ".env";
+
+            setup(() => {
+                fs.writeFileSync(envFile, envForFile);
+            });
+
+            teardown(() => {
+                fs.unlinkSync(envFile);
+            });
+
+            test("args from envFile should not overwrite existing variables", function() {
+                assert.deepEqual(GeneralMobilePlatform.getEnvArgument(origEnv, null, envFile), {
+                    "test1": "origEnv",
+                     "test2": "origEnv",
+                      "test3": "origEnv",
+                       "test4": "envFile",
+                        "test5": "envFile"});
+            });
+
+            test("args from envFile and original args should be overwrited by env args", function() {
```

Comment:
I think it makes sense to add one another test for case if `envFile` is null or undefined.